### ISSUE TITLE
PGSQL support?

### ISSUE THREAD

**Issue by salvoravida** [DTStack/monaco-sql-languages#15]
Hi, on the homepage, there is a mention of PGSQL support.

Where can I find the relative worker?

**Comment by wewoor** [comment]
> Hi, on the homepage, there is a mention of PGSQL support.
> 
> Where can I find the relative worker?

Sorry, it's a mistake of docs, I think you can try to compile the https://github.com/antlr/grammars-v4/tree/master/sql/postgresql to parser and integrate it by yourself.

**Comment by wewoor** [comment]
@salvoravida of course, you can refer to dt-sql-parser project, or use this https://github.com/tunnelvisionlabs/antlr4ts project

**Comment by salvoravida** [comment]
rc/grammar/pgsql/PostgreSQLLexer.g4:2196:29: syntax error: mismatched input 'options {' expecting COLON while matching a lexer rule

@wewoor I'm getting this error while building. Any idea? is it possible that the 4.8 ant version is old?

**Comment by wewoor** [comment]
@salvoravida https://www.npmjs.com/package/monaco-sql-languages/v/0.10.0
